Tasks/JenkinsQueueJobV2/job.ts

```diff
@@ -218,7 +218,7 @@ export class Job {
             return tl.TaskResult.Succeeded;
         } else if (this.State === JobState.Done) {
             const resultCode = this.ParsedExecutionResult.result.toUpperCase();
-            if (resultCode == 'SUCCESS' || resultCode == 'UNSTABLE') {
+            if (resultCode == 'SUCCESS' || (resultCode == 'UNSTABLE' && this.queue.TaskOptions.strictDefinition)) {
```

Comment:
Should not it be here `!this.queue.TaskOptions.strictDefinition`?

Reply:
> Please also include task-loc.json changes :)

Done

Reply:
Yes, my bad, supposed to be NOT